packages/web/modals/validator-squad.tsx

```diff
@@ -260,47 +249,55 @@ export const ValidatorSquadModal: FunctionComponent<ValidatorSquadModalProps> =
                 id: "validatorName",
                 accessorKey: "validatorName",
                 header: () => t("stake.validatorSquad.column.validator"),
-                cell: (
-                  props: CellContext<FormattedValidator, FormattedValidator>
-                ) => {
-                  const formattedWebsite = props.row.original.formattedWebsite;
-                  const website = props.row.original.website;
-
-                  return (
-                    <div className="flex max-w-[15.625rem] items-center gap-3 sm:w-[300px]">
-                      <div className="h-10 w-10 overflow-hidden rounded-full">
-                        <FallbackImg
-                          alt={props.row.original.validatorName}
-                          src={props.row.original.imageUrl}
-                          fallbacksrc="/icons/superfluid-osmo.svg"
-                          height={40}
-                          width={40}
-                        />
-                      </div>
-                      <div className="flex flex-col">
-                        <div className="subtitle1 md:subtitle2 text-left">
-                          {props.row.original.validatorName}
+                cell: observer(
+                  (
+                    props: CellContext<FormattedValidator, FormattedValidator>
+                  ) => {
+                    const formattedWebsite =
```

Comment:
Just a stylistic thing but you can do const { formattedWebsite } = props.row.original